stratum/hal/bin/tdi/tdi_main.cc

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018-2019 Barefoot Networks, Inc.
+// Copyright 2020-present Open Networking Foundation
+// Copyright 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "gflags/gflags.h"
+#include "stratum/glue/init_google.h"
+#include "stratum/glue/logging.h"
+#include "stratum/hal/lib/phal/phal_sim.h"
+#include "stratum/hal/lib/tdi/tdi_chassis_manager.h"
+#include "stratum/hal/lib/tdi/tdi_sde_wrapper.h"
+#include "stratum/hal/lib/tdi/tdi_action_profile_manager.h"
+#include "stratum/hal/lib/tdi/tdi_counter_manager.h"
+#include "stratum/hal/lib/tdi/tdi_hal.h"
+#include "stratum/hal/lib/tdi/tdi_node.h"
+#include "stratum/hal/lib/tdi/tdi_pre_manager.h"
+#include "stratum/hal/lib/tdi/tdi_switch.h"
+#include "stratum/hal/lib/tdi/tdi_table_manager.h"
+#include "stratum/lib/security/auth_policy_checker.h"
+
+DEFINE_string(tdi_sde_install, "/opt/sde",
+              "Absolute path to the directory where the SDE is installed");
+DEFINE_bool(tdi_switchd_background, false,
+            "Run switch daemon in the background with no interactive features");
+// TODO: Rename default configuration file.
+DEFINE_string(tdi_switchd_cfg, "stratum/hal/bin/tdi/tofino_skip_p4.conf",
+              "Path to the switch daemon json config file");
+
+namespace stratum {
+namespace hal {
+namespace barefoot {
```

Comment:
Same comment for all Tdi files also. Please remove barefoot.

Reply:
This appears to be a non-trivial change. I'm inclined to revisit the issue after the September release.